skimage/color/colorconv.py

```diff
@@ -679,6 +679,85 @@ def rgb2xyz(rgb):
     return _convert(xyz_from_rgb, arr)
 
 
+
+def rgb2c1c2c3(rgb):
+    """RGB to C1C2C3 color space conversion.
+
+    Parameters
+    ----------
+    rgb : array_like
+        The image in RGB format, in a 3-D array of shape ``(.., .., 3)``.
+
+    Returns
+    -------
+    out : ndarray
+        The image in C1C2C3 format, in a 3-D array of shape ``(.., .., 3)``.
+
+    Raises
+    ------
+    ValueError
+        If `rgb` is not a 3-D array of shape ``(.., .., 3)``.
+
+
+    References
+    ----------
+    .. [1] https://arxiv.org/abs/1702.05421
```

Comment:
This reference needs to be fleshed out some more. I wish we could convert bib files to our format easily.